src/prisma/engine/utils.py

```diff
@@ -29,20 +32,37 @@
 }
 
 
-def ensure() -> Path:
+def query_engine_name() -> str:
+    return f'prisma-query-engine-{platform.check_for_extension(platform.binary_platform())}'
+
+
+# TODO: detect this smarter, search for the current platform name?
+def _resolve_from_binary_paths(binary_paths: dict[str, str]) -> Path | None:
```

Comment:
I think the type of the `dict` here should be `Dict` (unless type checkers now allow this?)

Reply:
In Python 3.9+ or in Python 3.7+ with `from __future__ import annotations` you can use the actual runtime types in type hints like this.